irods-vfs-impl/src/main/java/org/irods/nfsrods/vfs/ServerMain.java

```diff
@@ -49,6 +50,14 @@ public class ServerMain
 
     public static void main(String[] args) throws JargonException, IOException
     {
+        String appName = "NFSRODS";
+        try {
+            StartupPack.setApplicationName(appName);
+        } catch (Exception e) {
+            log_.warn("main - Error setting application name [{}]", appName);
+        }
```

Comment:
If NFSRODS is running on a Jargon version prior to the new one that we release, shouldn't we need the try catch to make sure we're not trying to access a method that isn't there?

Or, does the `pom.xml` file make sure that something like this can't happen?